### PR TITLE
Feature aero common folder

### PR DESCRIPTION
**Pull request by planes**
Combining all the aero methods into a common folder. Deleting a few redundant scripts.

### CODE REVIEW

**Comment by timdmacdo** [comment]
It looks like the higher fidelity functions could be combined with the common ones. Why did you decide not to do that?

**Comment by planes** [comment]
Huh? They are all combined? What are you looking at?
https://github.com/suavecode/SUAVE/tree/feature-aero_common_folder/trunk/SUAVE/Methods/Aerodynamics/Common

Only if things are higher fidelity necessary are they in the other folders. You'll notice that there are no longer subsonic euler methods.

File: trunk/SUAVE/Methods/Aerodynamics/Common/Higher_Fidelity/Drag/untrimmed.py [hunk 1]
```diff
@@ -1,12 +1,14 @@
-## @ingroup Methods-Aerodynamics-SU2_Euler
+## @ingroup Methods-Aerodynamics-Higher_Fidelity-Drag
```
Comment (timdmacdo) [inline trunk/SUAVE/Methods/Aerodynamics/Common/Higher_Fidelity/Drag/untrimmed.py]: Higher fidelity untrimmed

File: trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Drag/untrimmed.py [hunk 2]
```diff
@@ -1,18 +1,14 @@
-## @ingroup Methods-Aerodynamics-Fidelity_Zero-Drag
+## @ingroup Methods-Aerodynamics-Common-Fidelity_Zero-Drag
```
Comment (timdmacdo) [inline trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Drag/untrimmed.py]: Common untrimmed

File: trunk/SUAVE/Methods/Aerodynamics/Common/Higher_Fidelity/Drag/untrimmed.py (hunk 1 above)
Comment (planes) [inline trunk/SUAVE/Methods/Aerodynamics/Common/Higher_Fidelity/Drag/untrimmed.py]: Do a compare of the texts. They are different enough to justify separate scripts.

File: trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Drag/untrimmed.py (hunk 2 above)
Comment (planes) [inline trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Drag/untrimmed.py]: Do a compare of the texts. They are different enough to justify separate scripts.

File: trunk/SUAVE/Methods/Aerodynamics/Common/Higher_Fidelity/Drag/untrimmed.py (hunk 1 above)
Comment (timdmacdo) [inline trunk/SUAVE/Methods/Aerodynamics/Common/Higher_Fidelity/Drag/untrimmed.py]: I did but in this one for example it looks like it's just a matter of when items are added.

**Comment by planes** [comment]
Ok, I've removed the high fidelity untrimmed scripts. I have strong words for whoever is responsible for coding that script that way. 

Also I noticed another VSP script duplicate.

**Comment by timdmacdo** [comment]
I think we can get rid of the other two files in higher fidelity too. I just ran some things to check so I can add those changes if you want.

**Comment by planes** [comment]
Sure if you think you can condense more while passing the regressions then go for it. You've done more with the aero models to know where there are synergies.

If we don't need higher fidelity then delete it.

**Comment by timdmacdo** [comment]
I'm good with @planes changes. I've added changes to remove the Higher_Fidelity common folder. Waiting on @mclarke2 for final approval.

**Comment by planes** [comment]
@timdmacdo do you think the function drag_div can be made common between VSP and Super-FZ?

**Comment by timdmacdo** [comment]
Looks like it. I think most of that compressibility method could be shared with a bit of work. I can go through all the leftover stuff in a bit more detail tomorrow and see if we can combine anything else.

**Pull request merged**